tests/test_ujson.py

```diff
@@ -533,6 +533,10 @@ def test_decode_numeric_int_exp(test_input):
     assert output == json.loads(test_input)
 
 
+def test_decode_long_float():
+    assert ujson.decode("100000000000000000000.0") == 1e20
```

Comment:
Please could you remove this test and add to the parametrized test at line 507?

Reply:
Done! :-)